frontend/packages/core/src/Resolver/fetch.tsx

```diff
@@ -1,15 +1,15 @@
 import type { clutch as IClutch } from "@clutch-sh/api";
-import * as $pbclutch from "@clutch-sh/api";
 import _ from "lodash";
 
 import { client, parseErrorMessage } from "../network";
 
 const fetchResourceSchemas = async (type: string): Promise<IClutch.resolver.v1.Schema[]> => {
+  const pbclutch = await require(`${process.env.REACT_APP_API_PACKAGE || "@clutch-sh/api"}`);
```

Comment:
This is great!

Can we read this in from the clutch config file instead of a process env to keep (mostly) everything in the same place?